### PR TITLE
update version 3.6.0

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,229 +1,317 @@
...
+    - lz4-c
     - openjpeg
     - openssl
-    - pcre
-    - poppler <=0.81  # [not win]
+    - pcre2
+    - poppler
     - postgresql
     - proj
+    - qhull
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:69]: There were noticed issues with qhull https://github.com/conda-forge/qgis-feedstock/issues/284#issuecomment-1356490896

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,229 +1,317 @@
...
+        - lz4-c
         - openjpeg
         - openssl
-        - pcre
-        - poppler <=0.81  # [not win]
+        - pcre2
+        - poppler
+        - postgresql
         - proj
+        - qhull
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:134]: There were noticed issues with `qhull` https://github.com/conda-forge/qgis-feedstock/issues/284#issuecomment-1356490896

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,229 +1,317 @@
...
         - {{ compiler('cxx') }}
-        - cmake  # [win]
-        - make   # [not win]
+        - cmake
+        - make   # [unix]
+        - ninja  # [win]
+
       host:
         - python
-        - setuptools <58.0.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:204]: Missing setuptools and wheel

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,229 +1,317 @@
...
-    script: install_python.sh   # [unix]
+    script: install_python.sh  # [unix]
     script: install_python.bat  # [win]
     build:
-      ignore_run_exports:
-        - openssl
-        - hdf5
+      skip_compile_pyc:
+        - share/bash-completion/completions/*.py
+
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:272]: Skip `py<36`, see https://github.com/OSGeo/gdal/blob/b77481ccf9f84a708fd37a13a7e0a3384fdf1622/swig/python/setup.py.in#L358

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,229 +1,317 @@
...
-    # Fix missing include
-    - patches/include-limits-missing.patch  # [linux]
+    - 0001-Improve-numpy-fixing.patch
+    - 0002-Parquet-make-sure-that-ArrowLayer-destructor-is-avai.patch
 
 build:
-  number: 6
-  # Has never been built for s390x
-  skip: True  # [s390x]
+  number: 14
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]: Build number must be `0`

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml:69]: yes, but I couldn't reproduce ... not sure here, but not sure how much we need to be concerned.  we do not provide qgis AFAIK, and common testsuite hasn't shown any regressions so far

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:134]: see above ^

File: recipe/meta.yaml (hunk 4 above)
Comment (katietz) [reply recipe/meta.yaml:273]: thanks added

File: recipe/meta.yaml (hunk 5 above)
Comment (katietz) [reply recipe/meta.yaml:15]: fair point

File: recipe/meta.yaml (hunk 3 above)
Comment (katietz) [reply recipe/meta.yaml:204]: add those dependencies. thanks for noticing

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**